src/VirtualClient/VirtualClient.Actions/FIO/FioDiscoveryExecutor.cs

```diff
@@ -52,6 +52,23 @@ public List<string> BlockSize
             }
         }
 
+        /// <summary>
+        /// The size of the test file that should use in workload tests (e.g. 496GB).
```

Comment:
Is this used anywhere?

Reply:
FileSize was inherited from DiskWorkloadExecutor (via FioExecutor) and used within FioDiscoveryExecutor. I removed the dependency between FioExecutor and DiskWorkloadExecutor, so I had to add this field here.
It is used in the ExecuteAsync method in DiskWorkloadExecutor.